src/components/QuickProjectSearch.tsx

```diff
@@ -201,28 +219,34 @@ export default function QuickProjectSearch() {
                       />
                     )}
 
-                    <div style={{ flex: 1 }}>
-                      <ProjectVersionBadge
-                        transparent
-                        size="small"
-                        versionText={`V${p.pv}`}
-                      />
+                    <div className="text-xs font-bold text-slate-200 dark:text-slate-200">
+                      <CurrencySymbol currency="ETH" />
+                      {formatWad(p.totalPaid, { precision: 0 })}
                     </div>
 
-                    {highlightIndex === i && <ArrowRightOutlined />}
+                    <ProjectVersionBadge
+                      className="text-slate-200 dark:text-slate-300"
+                      transparent
+                      size="small"
+                      versionText={`V${p.pv}`}
+                    />
+
+                    <div className="flex-1 text-right">
+                      {highlightIndex === i && <ArrowRightOutlined />}
+                    </div>
                   </div>
                 ))}
               </div>
             )}
 
-            {searchText && !isLoadingSearch && searchPages?.length === 0 && (
+            {searchText && !isLoadingSearch && searchResults?.length === 0 && (
               <div className="text-center text-grey-400">
                 <Trans>No results</Trans>
               </div>
             )}
           </div>
 
-          <div className="mt-5 flex gap-6 border-t border-r-0 border-b-0 border-l-0 border-solid border-t-smoke-200 bg-smoke-75 py-4 px-5 text-xs dark:border-t-slate-300 dark:bg-slate-600">
+          <div className="mt-5 flex gap-6 bg-smoke-75 py-3 px-5 text-xs dark:border-t-slate-300 dark:bg-slate-600">
```

Comment:
I think we still want the border here, wdyt? I feel like the colors blend in a weird way without it

**Without**
![Screen Shot 2023-01-27 at 12 55 39 pm](https://user-images.githubusercontent.com/104132113/214993884-faea5ee8-86e2-43f4-a572-0eff0970fc26.png)

**With**
![Screen Shot 2023-01-27 at 12 55 35 pm](https://user-images.githubusercontent.com/104132113/214993887-771770ce-083d-4b40-a001-71cc4ad96840.png)

Reply:
oh i'm not sure i meant to do this. though fwiw i'm team no-border, fewer lines look cleaner imo